fixtures/events/performance_problems/n-plus-one-in-django-index-view.json

```diff
@@ -168,7 +168,7 @@
       "same_process_as_parent": true
     },
     {
-      "timestamp": 1661959871.418989,
+      "timestamp": 1661957871.249481,
```

Comment:
Changed the timestamp here since it was being recognized as a Slow DB span and causing tests to fail